test/verify/check-machines-dbus

```diff
@@ -305,6 +307,7 @@ class TestMachinesDBus(machineslib.TestMachines):
             b.mouse("#vm-VmNotInstalled-firmware-tooltip", "mouseout")
             b.wait_not_present("#missing-uefi-images")
             m.execute("mv /tmp/ovmf/* {0}".format(ovmf_path))
+            self.addCleanup(m.execute, "rm -rf /tmp/ovmf")
```

Comment:
This looks wrong -- the previous command already moves all files in that dir into `ovmf_path`, and that already looks like a cleanup action of line 300. So maybe rather drop line 309 and replace it with an addCleanup immediately after the original mv on line 300?